ch_tools/chadmin/cli/zookeeper_group.py

```diff
@@ -41,8 +42,19 @@
     help="Do not try to get parameters from clickhouse config.xml.",
     default=False,
 )
+@option(
+    "-c",
+    "--chroot",
+    "zk_root_path",
+    type=str,
+    help="Cluster ZooKeeper root path. If not specified,the root path will be used.",
+    required=False,
+    default="/",
```

Comment:
It would be more convenient to try to auto-detect root path from ClickHouse config by default rather than have static default `"/`"`.